app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -25,10 +25,11 @@
         <blockquote v-if="storyReply" class="story-reply-quote">
           <span>{{ $t('CONVERSATION.REPLIED_TO_STORY') }}</span>
           <bubble-image
-            v-if="!hasStoryError"
+            v-if="!hasStoryError && storyUrl"
             :url="storyUrl"
             @error="onStoryLoadError"
           />
+          <bubble-video v-if="hasStoryError && storyUrl" :url="storyUrl" />
```

Comment:
@iamsivin Can we introduce the `v-else-if` section if Vue allows it, or does this work in a similar fashion?